flows/knack/knack_banner.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python
+
+"""
+Name: Knack Banner HR App integration
+Description: Update knack HR app based on records in Banner and CTM
+Schedule: "45 13 * * *"
+Labels: atd-data02, knack
+"""
+
+import os
+import docker
+import prefect
+from datetime import datetime, timedelta
+
+# Prefect
+from prefect import Flow, task
+from prefect.engine.results import PrefectResult
+from prefect.tasks.prefect import create_flow_run, get_task_run_result
+from prefect.storage import GitHub
+from prefect.run_configs import UniversalRun
+from prefect.engine.state import Failed, TriggerFailed, Retrying
+from prefect.schedules import Schedule
+from prefect.schedules.clocks import CronClock
+from prefect.backend import get_key_value
+from prefect.triggers import all_successful
+
+from prefect.utilities.notifications import slack_notifier
+from prefect.tasks.notifications.email_task import EmailTask
+
+current_environment = os.getenv("PREFECT_CURRENT_ENVIRONMENT", "production")
```

Comment:
Would this be an opportunity to define the "default" environment to be `staging` to have a fail-to-safe type of setup?

Reply:
the default environment usually is staging, i changed it here to be production because if it tries to run on staging its fails right out the gate.